commands/monitor.md

```diff
@@ -42,17 +42,17 @@ Manually issue the `QUIT` command to stop a `MONITOR` stream running via
 
 ## Commands not logged by MONITOR
 
-For security concerns, certain special administration commands like `CONFIG`
-are not logged into the `MONITOR` output.
+Because of security concerns, all administrative commands are not logged
+by `MONITOR`'s output.
 
-Other not logged commands: 
+Furthermore, the following commands are also not logged:
 
- * `AUTH` (start from Redis 6.0)
```

Comment:
@itamarhaber 
I reviewed the code seems logging `AUTH` is being removed from Redis 6.0, which version is in beta currently.
Right now 5.0 (and before) are more common versions, not sure if it will make confusion without indicate version.

Reply:
Are you saying that people will not scroll down all the way and read the "History" section?

Reply:
Oh, there it is, that's OK.

Remember that `EXEC` and `QUIT` are not excluded from 6.0, but from previous versions already.

Reply:
Right - fixed via 0bc9e22